persistence/data_models_json_fields.go

```diff
@@ -91,6 +91,164 @@ func (s *StateExecutionSequenceMapsJson) CompleteNewStateExecution(stateId strin
 	return nil
 }
 
+type StateExecutionWaitingQueuesJson struct {
+	// { state_execution_id_1: [ (queue_name_1, count_1), (queue_name_2, count_2), ... ], ... }
+	StateToCommandsMap map[string][]xdbapi.LocalQueueCommand `json:"stateToCommandsMap"`
```

Comment:
nit: `StateToCommandsMap`=> `StateToLocalQueueCommands` or `StateToLocalQueues` for shorter. 

Because I worry that in the future we may want to add something else like global queue, it will be confusing